Add release notes for couple of PRs (#979)
submariner-io/submariner-website: src/content/community/releases/_index.en.md

  environment which may forbid certain configurations in a disconnected Azure installation.
* `subctl` is now built for ARM Macs (Darwin arm64).
* `subctl show versions` now shows the versions of the metrics proxy and plugin syncer components.
* Submariner now uses case-insensitive comparison while parsing CNI names.

> * The `subctl gather` command now collects Metrics proxy pod logs in a Globalnet deployment.

Reviewer: ```suggestion
* The `subctl gather` command now collects metrics proxy pod logs in Globalnet deployments.
```

> * Submariner Gateway pod now skips invoking cableEngine cleanup during termination, as this is handled by the Route agent

Reviewer: ```suggestion
* Submariner gateway pods now skip cable engine cleanup during termination, as this is handled by the route agent
```

(not sure about the capitalisation...).

  during gateway migration.
* Fixed issue which caused the IPsec pluto process to crash when the remote endpoint was unstable.

> * Submariner now handles out-of-order remote endpoint notifications properly in Globalnet component.

Reviewer: ```suggestion
* The Globalnet component now handles out-of-order remote endpoint notifications properly.
```
